README.md

```diff
@@ -0,0 +1,31 @@
+![Logo](/Images/Logo.png)
+
+# Todoz*
+
+
+Todoz* is to-do projekt that will help you get things done in a way and manner you like! It's Weekly to-do planner (weeks are form sunday to sunday!) that will charm you with its funky and cute design, make you more productive by useful Notifications with descriptions that changes personality by your choosen *pussymeter* level.
+
+![Showcase](/Images/Showcase.png)
+
+#
+
+**Use our PussyMeter to personalizate your to-do!**
+
+Do you prefer more gental approach! *TodoZlatíčko* will be your kind friend.
+
+Want something causual? *TodoZápisník* is at your service!
+
+Need a kick in the ass, to do something? *TodoZ*rd* will be your personal productivity bully.
+
+We as TDZ team want Todoz* to be as much FUN and SIMPLE to use as posible, so could get rit of boredome that leats to procrastination!
```

Comment:
wtf bro is even this?
![Untitled](https://github.com/TDZ-production/todoz/assets/137875117/7d18653f-2485-4992-87fe-d2b5bdc347fd)